fixtures/stairville/xbrick-quad-16x8w-rgbw.json

```diff
@@ -0,0 +1,797 @@
+      "1": { "name": ["^1[A-D]$"] },
+      "2": { "name": ["^2[A-D]$"] },
+      "3": { "name": ["^3[A-D]$"] },
+      "4": { "name": ["^4[A-D]$"] }
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    }
+  },
+  "availableChannels": {
+    "Intensity red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Intensity green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Intensity blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Intensity white": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Stroboscope effect": {
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "helpWanted": "What are min/max strobe speeds?",
+        "speedStart": "0%",
+        "speedEnd": "100%"
+      }
+    },
+    "Function select": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 4],
+          "type": "NoFunction",
+          "switchChannels": {
+            "Intensity red / NoFunction": "Intensity red",
+            "Intensity green / NoFunction": "Intensity green",
+            "Intensity blue / NoFunction": "Intensity blue",
+            "Intensity white / NoFunction": "Intensity white",
+            "Dimmer / NoFunction": "NoFunction",
+            "Stroboscope effect / NoFunction": "NoFunction",
+            "NoFunction / Programme Speed / Microphone Sensitivity": "NoFunction"
+          }
+        },
+        {
+          "dmxRange": [5, 9],
+          "type": "ColorIntensity",
+          "color": "Cyan",
+          "switchChannels": {
+            "Intensity red / NoFunction": "NoFunction",
+            "Intensity green / NoFunction": "NoFunction",
+            "Intensity blue / NoFunction": "NoFunction",
+            "Intensity white / NoFunction": "NoFunction",
+            "Dimmer / NoFunction": "Dimmer",
+            "Stroboscope effect / NoFunction": "Stroboscope effect",
+            "NoFunction / Programme Speed / Microphone Sensitivity": "NoFunction"
+          }
+        },
+        {
+          "dmxRange": [10, 14],
+          "type": "Intensity",
+          "comment": "Purple",
```

Comment:
Some colours aren't available in `ColorIntensity` so I've just used `Intensity` and commented the colours. I noted that the default was 0% -> 100% so I removed the explicit `brightnessStart` and `brightnessEnd`.

Reply:
I think these should be `ColorPreset` capabilities rather than `ColorIntensity`/`Intensity` anyway, right?

Reply:
The manual says 'Intensity cyan (0 % to 100 %)', which suggests to me that the values 5 -> 9 map to Intensity 0% -> 100%. I agree though, there is very little granulaity in this and it may just be a colour preset.

Reply:
Maybe it's something like this?

```js
"type": "ColorPreset",
"colorsStart": ["#000000"],
"colorsEnd": ["#00ffff"]
```